styles.css

```diff
@@ -79,13 +83,27 @@ label {
   visibility: hidden;
 }
 
+#clear-btn {
+  font-family: 'Quicksand', sans-serif;
+  font-size: 16px;
+  background: #134d71;
+  color: #ffffff;
+  display: block;
+  border: 0;
+  border-radius: 6px;
+  width: 50px;
+  height: 28px;
+  align-self: baseline;
```

Comment:
I'm not sure `display: block` and 'align-self: baseline` are actually doing anything here. Once you get your button positioned correctly, it might be worth going back through the CSS and seeing if there's anything leftover you can delete.